php/tests/wrapper_type_setters_test.php

```diff
@@ -16,6 +16,44 @@
 
 class WrapperTypeSettersTest extends TestBase
 {
+    public function testConflictNormalVsWrapper()
```

Comment:
It looks like *both* functions have been renamed with a number on the end. Is that necessary? Why not just rename one of them, and keep the other unchanged?

Reply:
I think that is necessary. One field may be added before the other, and we don't want to accidentally change which field user is calling.

Consider the case if `int32 normal_vs_wrapper_value = 1;` is added first, and user is calling getNormalVsWrapperValue(), then someone added `google.protobuf.Int32Value normal_vs_wrapper = 2;`. If we rename the field 1 but not field 2, that user is suddenly reading from field 2, which is a behavior change. Similar problem exist for the other ordering too. Not to mention that it is also not clear which field getNormalVsWrapperValue() is referring to.